itest/lnd_channel_backup_test.go

```diff
@@ -1512,7 +1511,7 @@ func assertTimeLockSwept(ht *lntest.HarnessTest, carol, dave *node.HarnessNode,
 	// The commit sweep resolver publishes the sweep tx at defaultCSV-1 and
 	// we already mined one block after the commitment was published, and
 	// one block to trigger Carol's sweeps, so take that into account.
-	ht.MineEmptyBlocks(1)
+	ht.MineBlocks(2)
```

Comment:
hmm why do we need to mine more blocks eventho blockbeat should take care of the additonal block to trigger the sweep or ? I would expect to mine less blocks ?

Reply:
yeah the devil lies in the flow - because we removed multiple `MineBlocks` above, we now need to mine enough blocks to hit the timelock.